README.md

```diff
@@ -1,25 +1,147 @@
-# The Learn Philosophy
-
-# Welcome!!!!
+# Welcome to Learn
 
 Welcome to Learn! We're thrilled that you've joined our community of learners!
+In this lesson, we want to talk about the Learn platform and philosophy.
+
+The Learn platform, this web application with text to read, labs to move to
+"success," etc., is designed for people who are passionate, curious,
+self-driven, and serious about learning.
+
+We are comrades in a never-ending quest for knowledge. While there are no
+shortcuts, we believe that by providing knowledge in an optimized order,
+reinforcing knowledge with automated feedback from hands-on labs, and providing
+opportunities for discussion and feedback you will learn quickly. Learn is
+Flatiron School's framework for delivering that experience to you.  Our goal is
+provide you with the right materials, support and structure _you_ need to
+discover your own love for the material.
+
+## Building Birdhouses
+
+Imagine that we want to learn how to build a birdhouse but do not have any
+experience with carpentry. Where would we start? Is it necessary to _first_
+learn everything about the particular type of wood we are using? Or how nails
+and hammers are made?
+
+No. It may be _valuable_ to know what type of wood we're using, or _interesting_
+to understand how nails are made, but these things won't stop us from
+successfully building a birdhouse. In fact, we likely won't fully appreciate the
+materials, the wood and nails we're using until we've gone through the
+experience of _building the birdhouse first_.
+
+We call this approach "starting in the middle." We neither work bottom-up
+(explaining wood and nail manufacture as a first step) nor top-down (here's a
+bird house, take it apart and distill each concept until you reach an atomic
+idea). Instead we work from the middle: provide a relatable context, try it,
+try it again, break it down (a bit) and then explain how it integrates to the
+next idea. Let's imagine how this relates to birdhouses.
+
+After building birdhouse "Version 1," we can go back and better understand the
+best type of wood to use for our purposes, or realize that glue works just as
+well as nails and is much easier to use.  When we build birdhouse "Version 2,"
+we'll repeat the original again, but it'll be way better. By birdhouse "Version
+3," we'll know how everything fits together just by _looking_ at the materials.
+By "Version 15" we'll have developed a custom birdhouse building tool that,
+based on an input latitude / longitude submitted via smart phone, chooses the
+optimal wood for the bug populations and climates, cuts the wood, glues the
+edges, assembles and ships the birdhouse within 30 seconds of order.
+
+In this scenario, we will have become birdhouse magnates all without
+understanding one whit about nail manufacture, wood harvesting, or ornithology.
+
```

Comment:
I absolutely love this section. It's so precise and explains exactly our philosophy in a real world scenario, without being too obscure. It's just so good.